File: fix-group-node.js
const fs = require('fs');
const filePath = 'src/components/canvas/GroupNodeSimple.vue';
let content = fs.readFileSync(filePath, 'utf8');

content = content.replace(
  /const props = defineProps<{\s*id: string\s*data: unknown\s*selected\?: boolean\s*dragging\?: boolean\s*}>()/m,
  `const props = defineProps<{
  id: string
  data: Record<string, any>
  selected?: boolean
  dragging?: boolean
}>()`
);

fs.writeFileSync(filePath, content);

File: fix_group_node.py
import re

with open('src/components/canvas/GroupNodeSimple.vue', 'r') as f:
    content = f.read()

content = content.replace('data: unknown', 'data: Record<string, any>')
content = content.replace('props.data as Record<string, unknown> | undefined', 'props.data')

with open('src/components/canvas/GroupNodeSimple.vue', 'w') as f:
    f.write(content)
